app/assets/javascripts/i18n-strings.js.erb

```diff
@@ -3,6 +3,11 @@ window.LoginGov = window.LoginGov || {};
 window.LoginGov.I18n = {
```

Comment:
QQ -- does this file still need to be an ERB? I'm not seeing any `<%` inside it?

Reply:
Good point, probably not! IIRC, this file used to include all of what was later split to separate `fr`, `es`, and `en` scripts (the ones removed in this pull request).

I'll update.

Reply:
Aside: Eventually I'd like to remove this file altogether and replace it with a `packages/` module ([hinted at in one of the test cases below](https://github.com/18F/identity-idp/pull/5223/files#diff-d72e49f455e83fa1364669011350037fb850372ae5263f6c30693755337bcdafR22)).

Reply:
> QQ -- does this file still need to be an ERB? I'm not seeing any `<%` inside it?

Renamed in 5560978.